training/src/main/java/wolox/training/repositoies/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package wolox.training.repositoies;
+
+import java.util.List;
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
+    Book findFirstByAuthor(String author);
```

Comment:
A good practice is to return optional for better null pointer handling
```suggestion
    Optional<Book> findFirstByAuthor(String author);
```